src/runtime/nitro.ts

```diff
@@ -0,0 +1,16 @@
+import type { NitroAppPlugin, RenderResponse } from 'nitropack'
+import { useChecker, getValidator } from './validator'
+// @ts-expect-error virtual module
+import config from '#html-validator-config'
+
+export default <NitroAppPlugin> function (nitro) {
+  const validator = getValidator(config.options)
+  const { checkHTML } = useChecker(validator, config.usePrettier)
+
+  nitro.hooks.hook('render:response', (response: RenderResponse, { event }) => {
+    if (typeof response.body === 'string' && (response.headers['Content-Type'] || response.headers['content-type'])?.includes('html')) {
+      // We deliberately do not await response
```

Comment:
In cases where it's a positive addition to add a comment explaining special thoughts, it's always great to read *why* a decision was a good idea!

In this case I'd *guess* that not `await`ing improves performance in some way? :)

Reply:
Exactly. We don't want to block the rendering of the HTML but just provide lazy feedback in the console.